extensions/ql-vscode/src/interface.ts

```diff
@@ -262,10 +263,31 @@ export class InterfaceManager extends DisposableObject {
       sortedResultsMap,
       database: info.database,
       shouldKeepOldResultsWhileRendering,
-      kind: info.query.metadata ? info.query.metadata.kind : undefined
+      metadata: info.query.metadata
```

Comment:
Passing along all of the metadata here instead of just `kind` seems like a good idea, though.